recipe/meta.yaml

```diff
@@ -1,25 +1,27 @@
 {% set name = "tokenizers" %} 
-{% set version = "0.7.0" %}
+{% set version = "0.9.4" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/huggingface/tokenizers/archive/python-v{{ version }}.tar.gz
-  sha256: dc92c3f46fe03058c495663b1c125ce87722cad69167276465f9d2ce0284edf4
+  git_url: https://github.com/huggingface/tokenizers.git
+  git_rev: python-v{{ version }}
 
 build:
   number: 1
 
 requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('rust') }}
```

Comment:
Hmm.. Does this work?

Reply:
Yes on x86 it pulls -   ` rust_linux-64:          1.46.0-hf484d3e_0`